velox/functions/prestosql/aggregates/tests/VarianceAggregationTest.cpp

```diff
@@ -166,12 +166,17 @@ TEST_F(VarianceAggregationTest, varianceWithGlobalAggregation) {
 
   for (const auto& aggrName : aggrNames_) {
     // Global aggregation
+    // Post-aggregation projection addresses floating-point precision issues
```

Comment:
I'm not sure I understand what's happening here. We should be comparing using epsilon. Is this not working as expected? CC: @kagamiori

Reply:
No, it did not work for me. I found that `assertResults(sql)` did not trigger epsilon comparison but `assertResultsOrdered` did.